app/views/projects/contents.html.erb

```diff
@@ -51,5 +51,7 @@
 </div>
 <script type="module">
     // Setup the DataTables
```

Comment:
We should remove this comment, it's misleading (and probably a copy and paste thing)